nautobot/core/templates/generic/object_retrieve.html

```diff
@@ -85,12 +85,12 @@ <h1>{% block title %}{{ object }}{% endblock %}</h1>
                 </li>
             {% endif %}
         {% endif %}
-        {% if perms.extras.view_dynamicgroup and object.dynamic_groups %}
-            <li role="presentation"{% if request.GET.tab == 'dynamic-groups' %} class="active"{% endif %}>
-                <a href="{{ object.get_absolute_url }}#dynamic-groups" onclick="switch_tab(this.href)" aria-controls="advanced" role="tab" data-toggle="tab">
-                    Dynamic Groups
-                </a>
-            </li>
+        {% if perms.extras.view_dynamicgroup and object.get_dynamic_groups_url %}
+            {% if active_tab != 'dynamic-groups' and object.get_dynamic_groups_url or active_tab == 'dynamic-groups' %}
```

Comment:
Is this logic accurate? It reads funky to me.

Reply:
🤷‍♂️ Mimics what we are doing for changelog and notes tabs around it.